Geometry/CaloTopology/src/HcalTopology.cc

```diff
@@ -940,8 +929,7 @@ bool HcalTopology::decrementDepth(HcalDetId& detId) const {
       }
     }
   } else if ((subdet == HcalEndcap) && (etaRing == lastHERing()) && (depth == hcons_->getDepthEta29(iphi, zside, 0)) &&
-             (mode_ != HcalTopologyMode::SLHC) && (mode_ != HcalTopologyMode::Run3) &&
-             (mode_ != HcalTopologyMode::Run4)) {
+             !phase2()) {
```

Comment:
Again, `HcalTopologyMode::Run3` in not in your definition of `phase2()` as in [L175](https://github.com/cms-sw/cmssw/pull/46308/files#diff-bac5fd17f1d22b6df2fcd5469c58d31282ed972fc44d7f4c5c849c111c8b8441R175)

Reply:
Run3 is not a part of Phase2